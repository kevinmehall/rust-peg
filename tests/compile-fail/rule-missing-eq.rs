// We allow macro expansion to handle incomplete rules so rust-analyzer can complete a return type, but it should not compile
peg::parser!(grammar test() for str {
    rule test()
});

fn main() {}
