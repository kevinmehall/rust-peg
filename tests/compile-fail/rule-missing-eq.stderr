error: missing rule body
 --> tests/compile-fail/rule-missing-eq.rs:3:5
  |
3 |     rule test()
  |     ^^^^
